mock-data/MockData.ts

```diff
@@ -10,16 +10,16 @@ import type {
 
 const MockUpsertQuestions: QuestionUpsertDetail[] = [
 	{
-		id: 1,
-		number: 2,
+		id: 123,
+		number: 1,
 		questionText: 'It is blue and annoying, what is it?',
 		image: '/public/RiddleMeThis.jpeg',
 		hints: [],
-		correctAnswers: [{ text: 'Smurf' }]
+		correctAnswers: [{ text: 'SMURF' }]
```

Comment:
We will avoid unnecessary re-casting if we store correct answers already uppercase LIKE HERE.

Reply:
We will not avoid it because the re-casting will then be moved to the create riddle form.

Reply:
But it can be done only once ;)

Reply:
Ok, asi nechme být. Přišlo mi trošku zvláštní, že bych vytvořil otázku s odpovědí "Smurf". Celou hádanku i s touto otázkou bych uložil a pak se vrátil do editace a tam by bylo "SMURF".

Reply:
Na tom něco bude, asi bych zvolil, co bude jednodušší, můžeme kdyžtak později přepsat.